dataplicity/m2m/wsclient.py

```diff
@@ -112,6 +112,9 @@ def size(self):
     def __nonzero__(self):
         return self._data_event.is_set()
 
+    def __bool__(self):
```

Comment:
You may be right. I can't see it either. I may be confusing it for something I wrote.